docs/devsetup/java/maven.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: default
+title: Maven Setup for Java
+grand_parent: Setup for Contributors
+parent: Java Setup
+---
+
+<!--
+© 2016 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Maven Setup for Java
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+
+Maven is a pretty standard build tool in the Java ecosystem with a very well-defined
+preferred way to organize the directory structure and execute builds.
+Thus, many IDEs for Java will have some level of support for Maven.
+The command line invocation is still, of course, the standard of truth for the build.
+
+## Installation
+
+Install Maven from your OS package manager.
+The minimum version is 3.2.5, as configured in the `<ICU>/icu4j/pom.xml` root POM file's maven-enforcer-plugin.
```

Comment:
Do we want this level of detail?
This is the kind of stuff that gets outdated sooner or later.

---

Link to https://books.sonatype.com/mvnex-book/reference/installation.html ?

In general I would make this doc smaller and link to the relevant sections of "Maven by Example" or "Maven: The Complete Reference" as needed.

Reply:
Done.